src/libraries/Microsoft.PowerFx.Interpreter/Environment/SymbolValues.cs

```diff
@@ -12,19 +13,20 @@ namespace Microsoft.PowerFx
     /// Mutable collection for runtime Values for a <see cref="SymbolTable"/>.
     /// Will lazily create a symbol table. 
     /// To match to an existing symbol table, call <see cref="SymbolExtensions.CreateValues(ReadOnlySymbolTable, ReadOnlySymbolValues[])"/>.
-    /// </summary>
+    /// </summary>
+    [ThreadSafeImmutable]
```

Comment:
definitely not immutable - this has an Add method. 

See pattern in #1523 with [NotThreadSafe] in SymbolTable.cs